src/shogun/statistical_testing/internals/mmd/CrossValidationMMD.h

```diff
@@ -67,10 +67,11 @@ struct CrossValidationMMD : PermutationMMD
 		m_num_runs=DEFAULT_NUM_RUNS;
 		m_alpha=DEFAULT_ALPHA;
 
-		init();
+		init(seed);
```

Comment:
if seed = -1, then it's -1 and not a random seed, or?

Reply:
seed is not passed anymore. it's always a prng object that is passed, and for seeding created object, I added a function RandomNamespace called 'seed'